docs/examples.md

```diff
@@ -0,0 +1,99 @@
+examples that you can use to see how to write different algorithms.
+Most of these can be found in the
+[examples directory](https://github.com/quantumlib/Cirq/tree/master/examples)
+in the Github repository.
+
+In addition to these examples, you can also find the notebooks that generated
+the cirq docs found here in the
+[docs directory](https://github.com/quantumlib/Cirq/tree/master/docs) as well
+as all the more advanced case studies listed below this page in the table of
+contents.
+
+Below is a listing of the examples in the repository:
+
+## Beginning algorithms
+
+*    [Hello Qubit](https://github.com/quantumlib/Cirq/blob/master/examples/hello_qubit.py)
+Simple first program showing how to create a circuit.
+
+*    [Deutsch Algorithm](https://github.com/quantumlib/Cirq/blob/master/examples/deutsch.py)
+Textbook example of the simplest quantum advantage.
+
+*    [Bernstein-Vazirani](https://github.com/quantumlib/Cirq/blob/master/examples/bernstein_vazirani.py)
+Textbook algorithm determining properties of a hidden "oracle" function
+
+*    [Bell Inequality](https://github.com/quantumlib/Cirq/blob/master/examples/bell_inequality.py)
+Demonstration of a Bell inequality which shows violation of local hidden variables
+
+*    [BB84](https://github.com/quantumlib/Cirq/blob/master/examples/bb84.py)
+Textbook algorithm for Quantum key distribution
+
+*    [Noisy simulation](https://github.com/quantumlib/Cirq/blob/master/examples/noisy_simulation_example.py)
+     How to use a noisy simulator to generate results with amplitiude damping.
+
+*    [Line placement](https://github.com/quantumlib/Cirq/blob/master/examples/place_on_bristlecone.py)
+How to find a line of adjacent qubits on a device
+
+*    [Quantum Teleportation](https://github.com/quantumlib/Cirq/blob/master/examples/quantum_teleportation.py)
+A demonstration of using 2 classical bits to transport a quantum state from one
+qubit to another.
+
+*    [Super dense coding](https://github.com/quantumlib/Cirq/blob/master/examples/superdense_coding.py)
+     Transmit 2 classical bits using one quantum bit.
+
+
+# Intermediate Textbook Algorithms
+
+*    [Grover Algorithm](https://github.com/quantumlib/Cirq/blob/master/examples/grover.py)
+Example of applying the textbook Grover algorithm to find a 2-bit oracle
+function.
+
+*    [Quantum Fourier Transform](https://github.com/quantumlib/Cirq/blob/master/examples/quantum_fourier_transform.py)
+A demonstration of a 4-qubit quantum fourier transform (QFT)
+
+*    [Basic Arithmetic](https://github.com/quantumlib/Cirq/blob/master/examples/basic_arithmetic.py)
+Algorithms for adding and multiplying numbers as represented by qubit states
+
+*    [Phase estimation](https://github.com/quantumlib/Cirq/blob/master/examples/phase_estimator.py)
+     Textbook algorithm for phase estimation
+
+*    [Shor](https://github.com/quantumlib/Cirq/blob/master/examples/shor.py)
+Shor's algorithm for factoring.
```

Comment:
Maybe "Quantum algorithm for integer factoring." to avoid repeating Shor (see link on line above).